### ISSUE TITLE
4.5.3 release has Carthage error "Invalid architecture UUIDs"

### ISSUE THREAD

**Issue by rawrjustin** [adjust/ios_sdk#166]
when running Carthage update:
```
*** Downloading ios_sdk.framework binary at "Add bitcode flag"
Invalid architecture UUIDs: Could not parse UUIDs using dwarfdump from /private/var/folders/td/fqyg559527l6bzsz31_q1mkh0000gn/T/carthage-archive.DkT3w5/__MACOSX/AdjustSdkDynamic/AdjustSdk.framework.dSYM
```

Specifying an older release will work however i.e.
`github "adjust/ios_sdk" == 4.5.2`

**Comment by uerceg** [comment]
@rawrjustin Thanks for the notice. We'll investigate this. In the meantime, I have updated `carthage` branch on our repository with current master branch and it seems to work in there.

Can you please just change your `Cartfile` to point to `carthage` branch in order to get 4.5.3 version and let me know if it worked?
```
github "adjust/ios_sdk" "carthage"
```
In the meantime, we'll try to solve the issues which master branch has with Carthage.

**Comment by uerceg** [comment]
@rawrjustin We just released v4.5.4 with some improvements. Please check if Carthage is now successfully adding our SDK from `master` branch and let us know if it worked. Thanks in advance.

**Comment by rawrjustin** [comment]
Seems to be fixed in latest branches. 
 
`github "adjust/ios_sdk" "carthage"`
and 
`github "adjust/ios_sdk"` (master)

both seem to be working fine for me. 

Thanks @uerceg !
